apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/common/AbstractTransformerV2.java

```diff
@@ -0,0 +1,160 @@
+package gov.cms.bfd.server.war.r4.providers.preadj.common;
+
+import gov.cms.bfd.server.war.commons.IdentifierType;
+import gov.cms.bfd.server.war.commons.TransformerConstants;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.function.UnaryOperator;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Enumerations;
+import org.hl7.fhir.r4.model.HumanName;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Patient;
+import org.hl7.fhir.r4.model.StringType;
+
+// S1118 - Can't make constructor private because children won't be able to have one due to no
+// public constructor being defined.
+@SuppressWarnings("squid:S1118")
+public class AbstractTransformerV2 {
+
+  private static final Map<String, Enumerations.AdministrativeGender> GENDER_MAP =
+      Map.of(
```

Comment:
I just realized I didn't have the gender mapping into in the confluence page so I went ahead and updated that.  Its slightly different between FISS and MCS, to summarize

FISS male = "M"
FISS female = "F"
FISS unknown = "U"
MCS male = "M"
MCS female = "F"
MCS unknown = "O"

Note: neither FISS nor MCS use the "other" [administrative code](http://hl7.org/fhir/codesystem-administrative-gender.html)